create/node.go

```diff
@@ -207,7 +209,36 @@ func getBaseNodeTerraformConfig(terraformModulePath, selectedCluster string, sta
 		return baseNodeTerraformConfig{}, fmt.Errorf("Invalid rancher_host_label '%s', must be 'compute', 'etcd' or 'orchestration'", selectedHostLabel)
 	}
 
-	// TODO: Allow user to specify number of nodes to be created.
+	// Allow user to specify number of nodes to be created.
+	var countInput string
+	if viper.IsSet("node_count") {
+		countInput = viper.GetString("node_count")
+	} else {
+		prompt := promptui.Prompt{
+			Label: "Number of nodes to create",
+			Validate: func(input string) error {
+				_, err := strconv.ParseInt(input, 10, 64)
```

Comment:
Should we verify it's a positive number or zero?

Reply:
Yeah, good point. I'll fix that.